Engines/Wine/Plugins/Font smoothing/script.js

```diff
@@ -0,0 +1,35 @@
+include("engines.wine.engine.object");
+
+/**
+ * force the Font smothing
+ * @param {string} mode ("RGB", "BGR" or "Gray Scale")
+ * @returns {Wine} Wine object
+ */
+Wine.prototype.FontSmoothing = function (mode) {
```

Comment:
I recommend naming the method in Lowercase, i.e. `Wine.prototype.fontSmoothing`

Reply:
That's not in line wit the other plugin implementations then.

Reply:
Hmm, it looks like we're using sometimes lower case and sometimes upper case.
The `register font` plugin and the `nocrashdialog` plugin for example use lower case.

Reply:
Then let's open a new issue to:
- add a documentation page for verbs and plugins (including naming conventions)
- unify naming of the existing verbs and plugins

Reply:
See #1024, #1025 and #1026